components/auth-proxy-service/src/main/java/com/wso2telco/proxy/dao/attsharedaoimpl/AttShareDAOImpl.java

```diff
@@ -0,0 +1,122 @@
+package com.wso2telco.proxy.dao.attsharedaoimpl;
+
+
+import com.wso2telco.core.dbutils.DBUtilException;
+import com.wso2telco.core.dbutils.DbUtils;
+import com.wso2telco.proxy.dao.AttShareDAO;
+import com.wso2telco.proxy.util.AuthProxyEnum;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.identity.core.util.IdentityDatabaseUtil;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created on 9/20/17.
+ */
+public class AttShareDAOImpl implements AttShareDAO {
+
+    private static Log log = LogFactory.getLog(AttShareDAOImpl.class);
+
+    /**
+     * Close the database connection.
+     *
+     * @param connection Connection instance used by the method call
+     * @param statement  prepared Statement used by the method call
+     * @param resultSet  result set which is used by the method call
+     */
+    public void close(Connection connection, PreparedStatement statement, ResultSet resultSet) {
```

Comment:
No need of the method

Reply:
Removed close method and added 
`IdentityDatabaseUtil.closeAllConnections(connection, resultSet, preparedStatement);` to the code.